emails/models.py

```diff
@@ -32,13 +32,18 @@
 
 
 def get_domains_from_settings():
+    # HACK: detect if code is running in django tests
+    if 'testserver' in settings.ALLOWED_HOSTS:
+        return {
+            'RELAY_FIREFOX_DOMAIN': 'default.com',
+            'MOZMAIL_DOMAIN': 'test.com'
+        }
     return {
         'RELAY_FIREFOX_DOMAIN': settings.RELAY_FIREFOX_DOMAIN,
         'MOZMAIL_DOMAIN': settings.MOZMAIL_DOMAIN
     }
 
 
-DOMAINS = get_domains_from_settings()
```

Comment:
Testing this module-level variable was really nasty, so I removed it and replaced all uses with `get_domains_from_settings` instead. I also made `get_domains_from_settings` return mock/test values while in test mode.

Reply:
I was unsure about using conditionals for tests in feature code but I read some docs talked with @jwhitlock and this is a good approach. When he joins our team I hope we can talk about how we want to structure our test framework or our code so that code is more clear/concise and test conditionals can live in the test code.